src/Advertising.test.js

```diff
@@ -688,26 +850,51 @@ describe('When I instantiate an advertising main module without Prebid.js librar
 });
 
 // https://github.com/eBayClassifiedsGroup/react-advertising/issues/50
-describe(
-  'When I instantiate and initialize an Advertising module, ' +
-    'with gpt.js and prebid.js available, with config.usePrebid set to false',
-  () => {
-    let originalPbjs, originalGoogletag, advertising;
-    beforeEach(() => {
-      originalPbjs = setupPbjs();
-      originalGoogletag = setupGoogletag();
-      advertising = new Advertising({ ...config, usePrebid: false });
-      advertising.setup();
-    });
-    describe('the property `isPrebidUsed`', () =>
-      void it('is set to false', () =>
-        expect(advertising.isPrebidUsed).toBeFalsy()));
-    afterEach(() => {
-      global.pbjs = originalPbjs;
-      global.googletag = originalGoogletag;
+// eslint-disable-next-line max-len
+describe('When I instantiate and initialize an Advertising module, with gpt.js and prebid.js available, with config.usePrebid set to false', () => {
```

Comment:
Note to reviewers: This test case name used to be split across two lines, as two strings concatenated with `+`. However, we were unable to run it in VS code until we changed it to be one string with no concatenation. I don't know why it would be a problem, but if anyone has a better solution, please let us know!